gridlod/lod.py

```diff
@@ -304,10 +305,42 @@ def computeErrorIndicatorCoarseFromGreeks(patch, muTPrime, greeksPatch):
 
     deltaMaxTPrime, kappaMaxT = greeksPatch
 
-    epsilonTSquare = kappaMaxT**2 * np.sum((deltaMaxTPrime**2)*muTPrime)
+    epsilonTSquare = kappaMaxT**2 * np.sum((deltaMaxTPrime**2)*muTPrime) # TODO: is it really kappa**2? not kappa ?
 
     return np.sqrt(epsilonTSquare)
 
+def computeEftErrorIndicatorCoarseFromGreeks(etaT, cetaTPrime, greeksPatch):
+    '''Compute the coarse error idicator E(T) from the "greeks" delta and kappa,
+    where
+
+    deltaMaxTPrime = || ANew^{-1/2} (ANew - AOld) AOld^{-1/2} ||_max(TPrime)
+                                                     over all coarse elements TPrime in the patch
+
+    kappaMaxT = || AOld^{1/2) ANew^{-1/2} ||_max(T)
+                                 over the patch coarse T (patch.iPatchWorldCoarse)
+
+    nuT = || f - f_ref ||^2_L2(T)
```

Comment:
```suggestion
    nuT = || f - f_ref ||_L2(T)
```

I suggest we let nuT be without square (and also gammaT). Then it will look more like in the paper below